src/components/Points/index.tsx

```diff
@@ -51,13 +51,15 @@ const Points: React.FC<Props> = ({
       if (!func) {
         return Number.NaN;
       }
+      // @ts-ignore
       return +boundedSeries(xScale(func(d, i, arr)));
     });
 
     const [y, y0, y1] = [yAccessor, y0Accessor, y1Accessor].map(func => {
       if (!func) {
         return Number.NaN;
       }
+      // @ts-ignore
```

Comment:
same here?

Reply:
Need to rework typings, possibly Export both types and helpers from provider